docs/execution-providers/MIGraphX-ExecutionProvider.md

```diff
@@ -84,8 +85,60 @@ You can check [here](https://github.com/scxiao/ort_test/tree/master/python/run_o
 model on either the CPU or MIGraphX Execution Provider.
 
 
-## Configuration Options
-MIGraphX providers an environment variable ORT_MIGRAPHX_FP16_ENABLE to enable the FP16 mode.
+## Session Variables
+
+These flags can be invoked via the Provider Options struct [link] when creating an Onnxruntime Session Object and invoking the MIGraphXExecutionProvider 
+
+Items are added as a python dictionary when invoking the migraphx execution provider
+
+|Session Option Flag | Values | Description|
+|---|---|---|
+| device_id | INT | Selecit the device ID specified for the session run (default will be device 0) | 
+| migraphx_fp16_enable | 1 or 0 | Enable FP16 quantization mode via the MIGraphX API of the input model|
+| migraphx_int8_enable | 1 or 0 | Enable int8 static quantization mode of the input model via the MIGraphX API.\n Requires calibration table path vars to be set.|
+| migraphx_int8_calibration_table_name | <absolute path to calibration table> | Path to a set of input calibration data for int8 static model quantization |
+| migraphx_int8_use_native_calibration_table | 1 or 0 | Use a calibration table from Nvidia native int8 format or json dumped format|
+| migraphx_save_compiled_model | 1 or 0 | Enable saving a model as an MIGraphX (.mxr) format after compile |
+| migraphx_save_compiled_path  | <string> - Path to write .mxr file (default is ./compiled_model.mxr) | Path where the MIGraphX compiled model is stored. |
+| migraphx_load_compiled_model | 1 or 0 | Enable loading a model as an MIGraphX (.mxr) format after compile |
+| migraphx_load_compiled_path | <string> - Path to read .mxr file (default is ./compiled_model.mxr) | Path where the MIGraphX compiled model is stored. |
+| migraphx_exhaustive_tune | 1 or 0 (default 0) | Enable exhaustive tuning of parameters as part of compilation via the MIGraphX API. Adds additional compile time for a potential perf boost.|
```

Comment:
Aren't the other flags default 0? For consistency, I would just remove that part.
```suggestion
| migraphx_exhaustive_tune | 1 or 0 | Enable exhaustive tuning of parameters as part of compilation via the MIGraphX API. Adds additional compile time for a potential perf boost. |
```